### ISSUE TITLE
Syntax highlighting in VS Code fails because of missing NuGet.Frameworks

### ISSUE THREAD

**Issue by AntonRutkevich** [microsoft/qsharp-compiler#1495]
**Describe the bug**

Syntax highlighting in VS Code fails because of missing NuGet.Frameworks.

```
[Error - 5:32:36 PM] MSBuild error in /usr/local/share/dotnet/sdk/6.0.302/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(90,5): The "ProcessFrameworkReferences" task failed unexpectedly.
System.IO.FileLoadException: Could not load file or assembly 'NuGet.Frameworks, Version=6.2.1.7, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. Could not find or load a specific file. (0x80131621)
File name: 'NuGet.Frameworks, Version=6.2.1.7, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
```

Looks similar to [#434](https://github.com/microsoft/qsharp-compiler/issues/434) and [#86](https://github.com/microsoft/MSBuildLocator/issues/86). 

* Create local env: https://docs.microsoft.com/en-us/azure/quantum/install-command-line-qdk
* Add sample RNG: https://docs.microsoft.com/en-us/azure/quantum/quickstart-microsoft-qc?pivots=platform-ionq

**Expected behavior**

Syntax highlighting works out of the box after going through tutorial steps.

**Screenshots**

<img width="861" alt="Screenshot 2022-07-26 at 17 43 41" src="https://user-images.githubusercontent.com/14860034/181052465-7e6ae019-9578-4c30-9c63-6f0d7517cd8c.png">

**System information**

.Net, VSCode, Microsoft.Quantum.SDK are freshly installed.

- NuGet.Frameworks, Version=6.2.1.7, Culture=neutral, PublicKeyToken=31bf3856ad364e35
- OSX 12.4 (21F79); MacBook Pro (15-inch, 2016) 2,7 GHz Quad-Core Intel Core i7
- Microsoft.NET.Sdk: 6.0.302
- For issues with extensions, the version of the IDE for which the problem occurs 
- Microsoft.Quantum.Sdk/0.25.218240
- VSCode Version: 1.69.2, Commit: 3b889b090b5ad5793f524b5d1d39fda662b96a2a

**Additional context**

**Comment by bettinaheim** [comment]
@AntonRutkevich Could you please confirm with version of the VS Code extension you are using? We hope this should be fixed with the latest release (0.25.222597).

**Comment by AntonRutkevich** [comment]
Indeed, can't reproduce the issue with the now latest VS Code extension version (0.25.222597).
